### PR TITLE
[Model Monitoring] Fix model monitoring over feature store (httpSource)

### PR DESCRIPTION
**Pull request by katyakats**

### CODE REVIEW

File: mlrun/model_monitoring/helpers.py [hunk 1]
```diff
@@ -1,16 +1,68 @@
...
+    )
+
+    function.spec.graph = fset.spec.graph
+    function.spec.parameters = run_config.parameters
+    function.spec.graph_initializer = (
+        "mlrun.feature_store.ingestion.featureset_initializer"
+    )
+    function = http_source.add_nuclio_trigger(function)
+    function.metadata.credentials.access_key = model_monitoring_access_key
+    function.apply(mlrun.mount_v3io())
```
Comment (yaronha) [inline mlrun/model_monitoring/helpers.py:66]: @katyakats do we need this mount? i guess we can use the default v3io (object API) automount for IO

File: mlrun/model_monitoring/stream_processing_fs.py [hunk 2]
```diff
@@ -769,15 +781,4 @@ def get_endpoint_record(
...
-    parameters = json.loads(parameters) if parameters else {}
-    stream_processor = EventStreamProcessor(**parameters)
-    setattr(context, "stream_processor", stream_processor)
-
-
-def handler(context: MLClientCtx, event: Event):
-    event_body = json.loads(event.body)
-    context.logger.debug(event_body)
-    context.stream_processor.consume(event_body)
+# mlrun: end-code
```
Comment (yaronha) [inline mlrun/model_monitoring/stream_processing_fs.py:784]: not needed

**Comment by yaronha** [comment]
approved

File: tests/system/model_monitoring/test_model_monitoring.py [hunk 3]
```diff
@@ -221,22 +223,30 @@ def test_basic_model_monitoring(self):
         total = sum((m[1] for m in predictions_per_second.values))
         assert total > 0
 
-    @pytest.mark.timeout(200)
+    @pytest.mark.timeout(600)
     def test_model_monitoring_voting_ensemble(self):
-        simulation_time = 20  # 20 seconds
+        simulation_time = 120  # 60 seconds
```
[the remark this answers is not in the document]
Comment (katyakats) [reply tests/system/model_monitoring/test_model_monitoring.py:228]: to allow many events to be sent (if there are a few, the histogram created won;t be similar to the one from training)

**Pull request merged**